### DISCUSSION TITLE
ALAS本地部署失败，用CN与非CN均无效

### DISCUSSION THREAD

**Discussion by Loslinna** [LmeSzinc/AzurLaneAutoScript#2712] [Q&A] [answered]
昨天重装的系统，今天进行ALAS本地部署的时候失败

系统信息：Windows 11 专业版 22H2  操作版本22621.1778

![BUG-REPORT](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/121173540/d75e05d6-4fc7-44b3-a943-57adea82303d)

**Comment by ChenLily1019** [comment, marked as answer]
去看看安装教程，有关Repository那里
